doc/source/walkthrough.rst

```diff
@@ -37,20 +69,23 @@ To start a multi-node Ray cluster, see the `cluster setup page <using-ray-on-a-c
 Remote functions (Tasks)
 ------------------------
 
-Ray enables arbitrary Python functions to be executed asynchronously. These asynchronous Ray functions are called "remote functions". The standard way to turn a Python function into a remote function is to add the ``@ray.remote`` decorator. Here is an example.
+.. tabs::
+  .. group-tab:: Python
```

Comment:
This tab is pretty huge and contains a lot of duplicated text. Can we break it down into smaller tabs and have more common text shared between Python and Java?

Reply:
+1, Can you please make sure that every tab group is as small as it can possibly be?

Reply:
Updated